### PR TITLE
Update the Graph Service description

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #994 (Update graph service)

* Updates the Graph service description
(the text is sourced from the referenced issue/PR)
* Fixes a typo in a shared document
* Updates site.yml ocis rolling version, relevant for local build only

No backport

### CODE REVIEW

File: modules/ROOT/pages/deployment/services/s-list/graph.adoc [hunk 1]
```diff
@@ -42,6 +42,38 @@ The following image gives an overview of the scenario when a client requests to
...
+
+* *ldap*: +
+This is currently the default backend. It queries user and group information from an LDAP server. Depending on the configuration, it can also be used to manage (create, update, delete) users and groups provided by an LDAP server.
+
+* *cs3*: +
+This backend queries users and groups using the CS3 identity APIs as implemented by the `users` and `groups` service. This backend is currently still experimental and only implements a subset of the Libre Graph API. It should not be used in production.
+
+=== LDAP Configuration
+
+The LDAP backend is configured using a set of environment variables. A detailed list of all the available configuration options can be found in the xref:configuration[configuration section] below. The LDAP related options are prefixed with `OCIS_LDAP_` (or `GRAPH_LDAP_` for settings specific to graph service).
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/services/s-list/graph.adoc:57]:
```suggestion
The LDAP backend is configured using a set of environment variables. A detailed list of all the available configuration options can be found in the xref:configuration[configuration section] below. The LDAP related options are prefixed with `OCIS_LDAP_` (or `GRAPH_LDAP_` for settings specific to the graph service).
```

File: modules/ROOT/pages/deployment/services/s-list/graph.adoc [hunk 2]
```diff
@@ -42,6 +42,38 @@ The following image gives an overview of the scenario when a client requests to
...
+
+==== Read-Only Access to Existing LDAP Servers
+
+To connect the graph service to an existing LDAP server, set `OCIS_LDAP_SERVER_WRITE_ENABLED` to `false` to prevent the graph service from sending write operations to the LDAP server. Also set the various `OCIS_LDAP_*` environment variables to match the configuration of the LDAP server you are connecting to. An example configuration for connecting Infinite Scale to an instance of Microsoft Active Directory is available https://owncloud.dev/ocis/identity-provider/ldap-active-directory/[here,window=_blank].
+
+==== Using a Write Enabled LDAP Server
+
+To use the graph service for managing (create, update, delete) users and groups, a write enabled LDAP server is required. In the default configuration, the graph service will use the simple LDAP server that is bundled with infinite Scale in the xref:{s-path}/idm.adoc[idm] service which provides all the required features. It is also possible to setup up an external LDAP server with write access for use with Infinite Scale. It is recommend to use OpenLDAP for this. The LDAP server needs to fulfill a couple of requirements with respect to the available schema:
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/services/s-list/graph.adoc:65]:
```suggestion
To use the graph service for managing (create, update, delete) users and groups, a write enabled LDAP server is required. In the default configuration, the graph service will use the simple LDAP server that is bundled with infinite Scale in the xref:{s-path}/idm.adoc[idm] service which provides all the required features. It is also possible to setup up an external LDAP server with write access for use with Infinite Scale. It is recommended to use OpenLDAP for this. The LDAP server needs to fulfill a couple of requirements with respect to the available schema:
```

File: modules/ROOT/pages/deployment/services/s-list/graph.adoc [hunk 3]
```diff
@@ -42,6 +42,38 @@ The following image gives an overview of the scenario when a client requests to
...
+
+* The LDAP server must provide the `inetOrgPerson` object class for users and the `groupOfNames` object class for groups.
+
+* The graph service maintains a few additional attributes for users and groups that are not available in the standard LDAP schema. A schema file (ldif), ready to use with OpenLDAP defining those additional attributes, is available: {compose_url}v{compose_version}{compose_final_path}/ocis_ldap/config/ldap/schemas/[here,window=_blank].
+
+//https://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_ldap/config/ldap/schemas/10_owncloud_schema.ldif[here].
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/services/s-list/graph.adoc:71]: "and here"?

Are there 2 example schema files available?

File: modules/ROOT/pages/deployment/services/s-list/graph.adoc [hunk 4]
```diff
@@ -42,6 +42,38 @@ The following image gives an overview of the scenario when a client requests to
...
+
+== Query Filters Provided by the Graph API
+
+Some API endpoints provided by the graph service allow to specify query filters. The filter syntax is based on the https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_SystemQueryOptionfilter[OData Specification,window=_blank]. See the https://owncloud.dev/libre-graph-api/#/users/ListUsers[Libre Graph API,window=_blank] for examples on the filters supported when querying users.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/services/s-list/graph.adoc:75]:
```suggestion
Some API endpoints provided by the graph service allow query filters to be specified. The filter syntax is based on the https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_SystemQueryOptionfilter[OData Specification,window=_blank]. See the https://owncloud.dev/libre-graph-api/#/users/ListUsers[Libre Graph API,window=_blank] for examples on the filters supported when querying users.
```

File: modules/ROOT/pages/deployment/services/s-list/graph.adoc (hunk 3 above)
Comment (mmattel) [reply modules/ROOT/pages/deployment/services/s-list/graph.adoc:71]: No, I just forgot to remove the commented one which was a template to see if it renders correctly 😅 I will remove it now.

Comment (mmattel) [inline modules/ROOT/pages/deployment/services/s-list/graph.adoc:71]:
```suggestion
```
Removing commented example

Comment (mmattel) [reply modules/ROOT/pages/deployment/services/s-list/graph.adoc]: removed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**